process/pre_process/gtfs_stops_headway/gtfs_config.py

```diff
@@ -15,7 +15,7 @@
 headway_intervals = [20,30] # not implemented
 dissolve_cities = ['hanoi'] # aggregate mean of headways by stop_id; does not retain mode
 GTFS = {'maiduguri':[],
-		'hong_kong':[{'gtfs_filename': 'gtfs_input_data/gtfs_china_hongkong/gtfs_china_hongkong_hk_2019',
+		'hong_kong':[{'gtfs_filename': 'GTFS/gtfs_input_data/gtfs_china_hongkong/gtfs_china_hongkong_hk_2019',
```

Comment:
Just a minor thing here, may be not using the GTFS subfolder? Previously when I did the GTFS PR, I have modified the data folder to use the gtfs_input_data subfolder directly, not under the GTFS/. I felt like it would be more tidy this way...

Reply:
Hi @shiqin-liu, I get what you mean.  I suppose the reason I felt like it was useful is because it mirrors the existing data structure (eg. if the 'Global Indicators - Data' directory zip is downloaded from CloudStor it can be extracted directly as the 'process\data' folder), and it creates a seperation between the GTFS input data and associated files that might be located in the parent GTFS folder. 

So, for example, I have stored in this directory the output GTFS geopackages, a QGIS map document for some visual checking I did, other notes etc.  Those aren't required inputs, but they need somewhere to be stored; up to now that's been in this folder.  When I went to re-run things, I would have had to restructure the historical data directory which always makes me uneasy with dependencies (eg do we refer to this folder in other scripts?).  For me it works as is, but if you feel strongly about removing it we can remove the parent folder.

Reply:
Hi, Carl, it makes sense to me now why you are keep the GTFS folder.  I don't have other data files in that folder so maybe that's why I found it unnecessary when I tried to organize things. I don't have a strong opinion on this, so feel free to go with what you have. But we may also need to update the readme in the data folder to reflect this change.